paasta_tools/autoscaling/autoscaling_cluster_lib.py

```diff
@@ -734,3 +752,38 @@ def get_instances_from_ip(ip, instance_descriptions):
     :returns: list of instance description dicts"""
     instances = [instance for instance in instance_descriptions if instance['PrivateIpAddress'] == ip]
     return instances
+
+
+def get_autoscaling_info():
```

Comment:
Hmm, I initially had this function in `metastatus_lib` but thought it would live better here...

I see what you mean though. Maybe I should return a list of the `AutoscalingInfo` objects and then just create the `table` in `metastatus_lib`?